phlib/guisup.c

```diff
@@ -1212,20 +1224,22 @@ PPH_LAYOUT_ITEM PhAddLayoutItemEx(
         item->LayoutParentItem = item->LayoutParentItem->LayoutParentItem;
     }
 
-    item->LayoutParentItem->NumberOfChildren++;
+    item->LayoutParentItem->NumberOfChildren += 1;
```

Comment:
🤔.. please revert this change.

Reply:
whats wrong? this is for correct layout manager work for for different dpi.